Dockerfile

```diff
@@ -1,22 +1,43 @@
 FROM golang:1.22-alpine3.19 AS builder
 
-RUN apk add --no-cache make
+# Create a non-root user
+RUN addgroup -g 1001 -S iamgroup && \
+	adduser -u 1001 -S iamuser -G iamgroup
```

Comment:
What is `iamuser`?

Reply:
`iamuser` is a non-root user, basically the purpose of creating this is so that we don't have to run the application as a root user.

Reply:
Yeah, but why `iamuser` :D funny name. The standard practice is to call the user the same name as the service, so just `telemetry`.